tests/IceRpc.Protobuf.Tests/OperationTests.cs

```diff
@@ -11,14 +11,24 @@ namespace IceRpc.Protobuf.Tests;
 public partial class OperationTests
 {
     [Test]
-    public void Unary_rpc()
+    public async Task Unary_rpc()
     {
         // Arrange
         var invoker = new ColocInvoker(new MyOperationsService());
         var client = new MyOperationsClient(invoker);
 
-        // Act/Assert
-        Assert.That(async () => await client.UnaryOpWithEmptyParamAndReturnAsync(new Empty()), Throws.Nothing);
+        var m = new InputMessage()
```

Comment:
I dont think we should use single letter variable names even in tests.

```suggestion
        var message = new InputMessage()
```